packages/core/src/index.js.flow

```diff
@@ -83,10 +83,6 @@ declare export function sample <A, B, C> (f: (A, B) => C, sampler: Stream<A>): (
 declare export function sample <A, B, C> (f: (A, B) => C): (sampler: Stream<A>) => (sampled: Stream<B>) => Stream<C>
 
 declare export function combine <A, B, Z> (f: (A, B) => Z, sa: Stream<A>, sb: Stream<B>): Stream<Z>
```

Comment:
Need to add the other partial application variants for combine and zip.  Take a look at [the scan type defs for an example](https://github.com/mostjs/core/blob/master/packages/core/src/index.js.flow#L43-L46).